automation/e2e-upgrade-functests/run.sh

```diff
@@ -20,9 +20,22 @@ NAMESPACE=${1:-kubevirt}
 RELEASE_BRANCH=${CI_BRANCH}
 if [[ -z ${RELEASE_BRANCH} ]] || [[ ${RELEASE_BRANCH} == "main" ]]
 then
-  # Get the latest release branch
-  RELEASE_BRANCH=$(curl -s 'https://api.github.com/repos/kubevirt/ssp-operator/branches?per_page=100' |
-    jq -r '[.[].name | select(startswith("release-v"))] | max_by(ltrimstr("release-v") | split(".") | map(tonumber))')
+  PAGE=1
+  BRANCHES="[]"
+  DATA=$(curl -s "https://api.github.com/repos/kubevirt/ssp-operator/branches?page=$PAGE&per_page=100")
+  while [[ $(echo "$DATA" | jq 'length') -ne 0 ]]
+  do
+    BRANCHES=$(jq -s 'add' <(echo "$BRANCHES") <(echo "$DATA"))
+    ((PAGE++))
+    DATA=$(curl -s "https://api.github.com/repos/kubevirt/ssp-operator/branches?page=$PAGE&per_page=100")
+  done
```

Comment:
```suggestion
while true; do
  DATA=$(curl -s "https://api.github.com/repos/kubevirt/ssp-operator/branches?page=$PAGE&per_page=100")
  if [[ $(echo "$DATA" | jq 'length') -eq 0 ]]; then
    break
  fi
  BRANCHES=$(jq -s 'add' <(echo "$BRANCHES") <(echo "$DATA"))
  ((PAGE++))
done
```

Avoids declaring DATA twice :)